src/sagemaker_xgboost_container/serving.py

```diff
@@ -43,6 +43,13 @@ def is_multi_model():
     return os.environ.get("SAGEMAKER_MULTI_MODEL")
 
 
+def set_default_env():
+    env_default_dict = {"OMP_NUM_THREADS": "1"}
```

Comment:
Please define `"1"` as a constant somewhere with a descriptive variable name.

Reply:
its optimization of multiprocessing parameters since we are tuning threads per process given that process count normally equals CPU core count, but yes we should try to be more descriptive here of the motivation so that someone outside this conversation can see what's going on here. It needs to be standalone in understanding.